contrib_bots/lib/commute_bot.py

```diff
@@ -0,0 +1,262 @@
+import datetime as dt
+import requests
+from __future__ import print_function
```

Comment:
This needs to move to the top of the file--it's what's causing the latest Travis errors.